### ISSUE TITLE
d2l-pager-load-more not accessible/focusable in d2l-list with grid attribute

### ISSUE THREAD

**Issue by ChrisLabattD2L** [BrightspaceUI/core#3034]
Currently you cannot access the `d2l-pager-load-more` via keyboard navigation when it's being used in association with a `d2l-list` that has the `grid` attribute.

The only work-around for this is if there are other focusable components after the list & load-more button. In that case, you can tab to that, then reverse tab into the `d2l-pager-load-more` component. This isn't much a workaround though, since you'd need to have a design with this in mind, which isn't always possible.

**Comment by dbatiste** [comment]
Thanks for raising this Chris. We should be able to make a fix for this.

**Comment by ChrisLabattD2L** [comment]
Thanks Dave! I've noticed that there's an instance in which this issue doesn't occur, which is when the last item in the list isn't focusable due to not having a `href`. I'm not sure if that's related to the problem or not, but I figure I'd mention it in case it's relevant.

**Comment by svanherk** [comment]
I've created [DE51137](https://rally1.rallydev.com/#/15545167705ud/custom/21568985922?detail=%2Fdefect%2F672337163059) if you want to follow along!

**Comment by ChrisLabattD2L** [comment]
Thanks Stacey!

**Comment by svanherk** [comment]
This should be fixed for `core` version `2.67.4`!  @ChrisLabattD2L let us know if you still see any issues!

**Comment by ChrisLabattD2L** [comment]
Just tested it quickly, and it's no longer an issue, thanks 😄
